src/install/deploySteps.js

```diff
@@ -150,25 +157,26 @@ module.exports = (apps, mode, deployDoc) => {
       });
   };
 
-const updateSymlinkAndRemoveOldVersion = changedApps =>
-  Promise.all(changedApps.map(app => {
-    const livePath = deployPath(app, 'current');
+  const updateSymlinkAndRemoveOldVersion = changedApps => {
+    return Promise.all(changedApps.map(app => {
+      const livePath = deployPath(app, 'current');
 
-    if(fs.existsSync(livePath)) {
-      const linkString = fs.readlinkSync(livePath);
+      if(fs.existsSync(livePath)) {
+        const linkString = fs.readlinkSync(livePath);
 
-      if(fs.existsSync(linkString)) {
-        debug(`Deleting old ${app.name} from ${linkString}…`);
-        fs.removeSync(linkString);
-      } else debug(`Old app not found at ${linkString}.`);
+        if(fs.existsSync(linkString)) {
+          debug(`Deleting old ${app.name} from ${linkString}…`);
+          fs.removeSync(linkString);
+        } else debug(`Old app not found at ${linkString}.`);
 
-      fs.unlinkSync(livePath);
-    }
+        fs.unlinkSync(livePath);
+      }
 
-    fs.symlinkSync(deployPath(app), livePath);
+      fs.symlinkSync(deployPath(app), livePath);
 
-    return Promise.resolve();
-  }));
+      return Promise.resolve();
```

Comment:
I'm pretty sure this is unnecessary (although with the indent changes it's not 100% clear). Because it's wrapped in a `Promise.all` returning nothing at all is the same as returning a `Promise.resolve()`.

Reply:
It looks unnecessary to me as well (I didn't write it I just changed the style to the one you want with the `{}` wrapping functions)

Reply:
I noticed and appreciated it!